test/input-states-mocha.js

```diff
@@ -0,0 +1,341 @@
+/**
+ * File: input-state-mocha.js
+ * Unit tests for the input-state APIs defined in src/input-state.js
```

Comment:
Does this test for the following cases?

 - An optional port that is not attached to anything should return an undefined state.

 - A multi port with nothing attached should return an empty list of states.

 - Ensure that required ports are attached and check for too many attachments to non-multi ports ( https://github.com/rdf-pipeline/noflo-rdf-pipeline/issues/24 ).  These checks do not necessarily have to be done in this module though.

 - Getting the states from a multi port with both an IIP and an edge attached, with a vnid of '' coming on the edge attachment.

 - Getting the states from a multi port with both an IIP and an edge attached, with a non-empty-string vnid coming on the edge attachment.